packages/mux-player/src/media-theme-mux.html

```diff
@@ -53,8 +53,8 @@
       width: 100%;
     }
 
-    [disabled],
-    [aria-disabled='true'] {
+    [disabled]:not(media-live-button),
+    [aria-disabled='true']:not(media-live-button) {
```

Comment:
NOTE: Unrelated fix for styling of live button (which is disabled when at the live edge)